java/robox-slicer-extension/robox-slicer-flow/src/main/java/com/roboxing/slicerextension/flow/Slicer.java

```diff
@@ -68,6 +68,8 @@ public JSONObject getSlicerConfig() {
     }
 
     public void postProcess(File inputGCode, File resultGCode) throws IOException{
+        String newLine = System.lineSeparator();
```

Comment:
OK - now I've checked more of the code. No need for newLine at all! :) 

Everywhere where you have outputRelative just use println and it will do the new line accordingly to OS. We are using PrintWriter - https://docs.oracle.com/javase/8/docs/api/java/io/PrintWriter.html